husarion_ugv_hardware_interfaces/src/robot_system/ugv_system.cpp

```diff
@@ -538,12 +544,11 @@ void UGVSystem::MotorsPowerEnable(const bool enable)
       }
     }
 
-    e_stop_->TriggerEStop();
-
     roboteq_error_filter_->SetClearErrorsFlag();
     roboteq_error_filter_->UpdateError(ErrorsFilterIds::ROBOTEQ_DRIVER, false);
   } catch (const std::runtime_error & e) {
-    RCLCPP_WARN_STREAM(logger_, "An exception occurred while enabling motors power: " << e.what());
+    RCLCPP_WARN_STREAM(
+      logger_, "An exception occurred while changing the torque state of the motors: " << e.what());
   }
```

Comment:
Not sure but I think this should additionally throw exception, so service can return message with error